baselines/FedSMOO/script.sh

```diff
@@ -0,0 +1,2 @@
+export HYDRA_CONFIG_PATH=conf/fedavg.yaml
+HYDRA_FULL_ERROR=1 python -m FedSMOO.main --config-name=fedavg ++num_rounds=800
```

Comment:
```suggestion
HYDRA_FULL_ERROR=1 python -m fedsmoo.main --config-name=fedavg num_rounds=800
```

Reply:
just highlighting to update this once your rename everything from `FedSMOO` to `fedsmoo`. Also, `num_rounds` doesn't need the `++`. Or am I missing something?

Reply:
Another comment regarding `--config-name` usage here. Since your default `config_name` is already `"fedavg"` in your `main.py`, there's no need make use of the `--config-name` flag. However you'll need it if you want to use your `fedsmoo` config.